### DISCUSSION TITLE
Skin Names Changing

### DISCUSSION THREAD

**Discussion by gamerturtlepoggers** [ppy/osu#16016] [Ideas]
Whenever I transfer my skins over from stable to lazer, the names of the skins change and it makes it really hard for me to find the skin I want. Is there an option that can turn this off/can be added?

I'm on an m1 mac

**Comment by Theighlin** [comment]
This was done because some skins have the same name in the skin.ini file and lazer couldn't import them both as all skins after the first one were read as already imported iirc. To avoid this, skin names have been coupled with their folder name which is bound to be different.

There will probably be the ability to change their name in the future

**Comment by peppy** [comment]
The names after import should contain the old name you're looking for. Is your issue that the order has changed in the list?
